pandas/tests/test_series.py

```diff
@@ -6449,21 +6449,122 @@ def test_apply_dont_convert_dtype(self):
         result = s.apply(f, convert_dtype=False)
         self.assertEqual(result.dtype, object)
 
-    # GH 10265
     def test_convert_objects(self):
+
+        s = Series([1., 2, 3], index=['a', 'b', 'c'])
+        result = s.convert_objects(convert_dates=False, convert_numeric=True)
```

Comment:
so anytime you use ``.convert_objects`` it needs a:

```
tm.assert_produces_warning(FutureWarning):
   .....
```

around, e.g. no need to show the deprecation warning.